jekyll/_cci2/orb-concepts.md

```diff
@@ -248,6 +244,47 @@ _[See: Writing Inline Orbs]({{site.baseurl}}/2.0/reusing-config/#writing-inline-
 - Not public
 - Not accessible via CircleCI CLI
 
+## Private orbs vs. public orbs
+{: #private-orbs-vs-public-orbs }
+
+There are two ways to publish an orb: public or private:
+
+* If you prefer to publish your orb so that only those within your organization can see and use it, you should publish a private orb.
+* If you want to publish your orb to the [CircleCI Orb Registry](https://circleci.com/developer/orbs) for use by anyone, create a public orb.
+
+Private orbs are described in more detail below.
+
+### Private orbs
+{: #private-orbs }
+
+**Note:** _Private orbs are available on any of our [plans listed on our plans page](https://circleci.com/pricing)._
+{: class="alert alert-warning"}
+
+Using a private orb enables you to author an orb while ensuring the following:
+
+* Your orb will not appear in the [CircleCI Orb Registry](https://circleci.com/developer/orbs) unless you have the direct URL and are authenticated to the org that created it.
```

Comment:
Authenticated _with_ rather than _to_?

Reply:
Oh yeah I was going to ask about that one, thought it sounded off